test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/architecture/UnmanagedKieServersWithSmartRouterIntegrationTest.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright 2017 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.cloud.integrationtests.architecture;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.kie.cloud.api.DeploymentScenarioBuilderFactory;
+import org.kie.cloud.api.deployment.constants.DeploymentConstants;
+import org.kie.cloud.api.scenario.GenericScenario;
+import org.kie.cloud.api.settings.DeploymentSettings;
+import org.kie.cloud.common.provider.KieServerClientProvider;
+import org.kie.cloud.common.provider.SmartRouterAdminClientProvider;
+import org.kie.server.api.model.instance.TaskSummary;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.client.ProcessServicesClient;
+import org.kie.server.client.QueryServicesClient;
+import org.kie.server.client.UserTaskServicesClient;
+import org.kie.server.integrationtests.router.client.KieServerRouterClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Test for Architecture 4 from
+ * http://mswiderski.blogspot.cz/2017/08/cloud-runtime-architectures-for-jbpm.html
+ */
+public class UnmanagedKieServersWithSmartRouterIntegrationTest extends AbstractCloudArchitectureIntegrationTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(UnmanagedKieServersWithSmartRouterIntegrationTest.class);
+
+    private final String RANDOM_URL_PREFIX = UUID.randomUUID().toString().substring(0, 4) + "-";
+
+    private static final String SMART_ROUTER_HOSTNAME = SMART_ROUTER_NAME + DeploymentConstants.getDefaultDomainSuffix();
+
+    private DeploymentSettings smartRouter;
+
+    private DeploymentSettings kieServerABC;
```

Comment:
(stricly optional) the same comment as above and throughout this PR for repeated type declarations. I find 
```
private DeploymentSettings
    kieServerABC,
    kieServerDEF,
    kieServerGHI;
```
takes some unnecessary verbosity away and makes it easier on the eyes, but stick with your version if you like it better.